tests/test_io.py

```diff
@@ -1095,6 +1095,7 @@ def test_read_sql_via_sqlalchemy_connection():
                 assert_type(read_sql("select * from test", con=conn), DataFrame),
                 DataFrame,
             )
+        engine.dispose()
```

Comment:
Can `engine` be used as a context manager? Might be slightly nicer if that's possible

Reply:
No, it is not possible.